civis/civis.py

```diff
@@ -338,10 +342,17 @@ def __init__(self, api_key=None, return_type='snake',
         self._session_kwargs = {'api_key': session_auth_key,
                                 'max_retries': retry_total}
 
-        classes = generate_classes_maybe_cached(local_api_spec,
-                                                session_auth_key,
-                                                api_version,
-                                                resources)
+        # Catch deprecation warnings from generate_classes_maybe_cached and
+        # the functions it calls until the `resources` argument is removed.
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                category=FutureWarning,
+                module='civis')
+            classes = generate_classes_maybe_cached(local_api_spec,
+                                                    session_auth_key,
+                                                    api_version,
+                                                    resources)
```

Comment:
Is this an interim fix?  Why not set a default argument to all of the functions with `resources` like `parse_api_spec` and `generate_classes_maybe_cached` and then in functions like this we simply don't pass along the `resources` parameters?  I don't believe that's not a breaking change since `resources='base'` is a subset of `resources='all'`?

Reply:
I was thinking we could just take it out when we do 2.0 rather than fiddling with it now and then taking it out later.

Reply:
Ah, okay I see what you meant above now